packages/nimble-components/src/theme-provider/index.ts

```diff
@@ -33,19 +54,44 @@ export const theme = DesignToken.create<Theme>({
  * @internal
  */
 export class ThemeProvider extends FoundationElement {
-    @attr({
-        attribute: 'direction'
-    })
+    @attr()
+    public override lang = '';
```

Comment:
I want `lang` to initially be `undefined`, i.e. no override of the `lang` design token's default. However I can't type it as `string | undefined` because `HTMLElement`'s `lang` property is typed as just `string`. The best I could think to do was to initialize to `''` and just treat that value the same as `null`/`undefined` in my `langChanged` handler. This seems reasonable, because `''` is otherwise an invalid value -- if we pass that to the `Intl.DateTimeFormat` constructor, it throws an error.